examples/generative/gpt2_text_generation_with_kerasnlp.py

```diff
@@ -25,16 +25,24 @@
 """
 
 """
-## Install KerasNLP and Import Dependencies
+## Install KerasNLP, Choose Backend and Import Dependencies
+
+We've written this example to run on top of Keras Core and it works with
```

Comment:
I would rephrase slightly...

The following example uses [Keras Core](https://keras.io/keras_core/) to work in any of `tensorflow`, `jax` or `torch`. Support for Keras Core is baked into KerasNLP, simply change the `KERAS_BACKEND` environment variable below to change the backend you would like to use. We select the `jax` backend below.